drop-in/src/main/java/com/adyen/checkout/dropin/service/DropInServiceResult.kt

```diff
@@ -81,6 +82,11 @@ sealed class DropInServiceResult : BaseDropInServiceResult() {
         override val reason: String? = null,
         override val dismissDropIn: Boolean = false
     ) : DropInServiceResult(), DropInServiceResultError
+
+    /**
+     * For internal use only.
+     */
```

Comment:
If it's for internal use only I don't think we need to add a comment just explaining that.

Reply:
hmm yeah fair point